modules/end-user-guide/pages/enabling-maven-artifact-repositories.adoc

```diff
@@ -82,6 +82,8 @@ data:
     </settings>
 ----
 
+. Optional: When using EAP-based devfiles, apply a second `settings-xml` configmap with the same content, a different name, and the `/home/jboss/.m2` mount path.
```

Comment:
```suggestion
. Optional: When using EAP-based devfiles, apply a second `settings-xml` ConfigMap with the same content, a different name, and the `/home/jboss/.m2` mount path.
```
I found this entry for "ConfigMap" in `contributing_to_docs/term_glossary.adoc`:
```
=== ConfigMap

Usage: ConfigMap(s)

ConfigMaps hold configuration data for Pods to consume.

Do not use: configuration map(s)
```

Reply:
Thank you, Rolfe.